splat.yaml

```diff
@@ -27,7 +30,6 @@ segments:
     subsegments:
     - [0x1000, hasm, entry]
     - [0x1060, c, sys/main]
-    - [0x1510, asm, padding_1510]
     - [0x1520, c, sys/sched]
```

Comment:
The .text section of `sys/sched` is 0x20 aligned. That's why there's padding at 0x1510
This means we can remove the 0x1510 entry but we need to change the `subalign` to `null`.

Reply:
Sadly this is the only padding that exists because of big alignment of the next file.